handlers/deposit_file.go

```diff
@@ -69,15 +69,8 @@ func (d *depositFileEntry) createFileResource(resourceID string, filename string
 	return d.database.Insert(resource)
 }
 
-func (d *depositFileEntry) buildPersistableResource(resourceID string, filename string) interface{} {
-	return map[string]interface{}{
-		"id": resourceID,
-		// TODO: Where should Access come from/default to?
-		"access":    "private",
-		"atcontext": atContext,
-		"attype":    fileType,
-		"label":     filename,
-		"preserve":  false,
-		"publish":   false,
-	}
+func (d *depositFileEntry) buildPersistableResource(resourceID string, filename string) db.Resource {
+	// TODO: Expand here if we need to set any default properties on the file
+	identification := map[string]interface{}{"filename": filename, "identifier": resourceID, "sdrUUID": resourceID}
```

Comment:
this will change pretty radically i expect, but i'm not sure its worth stopping this PR for right now. It would be brought up in #303